### ISSUE TITLE
Add more getters to staking providers smart contracts for testing purposes

### ISSUE THREAD

**Issue by itaven** [lidofinance/lido-dao#38]
### Suggestions: 
 **1)** Implement **getSigningKeys() function** which will return an array of signing keys that were added for current sp.
_Current solution_: right now we can only getSigningKey(sp_id,SigningKey_index)
_Reason_: for testing purposes where we will test the whole system with a big number of signing keys. We must check the correctness of added signing keys with the source. We can't expect how the system will react to it and also it's not too flexible to call getSigningKey(sp_index,signingKey_index) 200 times for example.
**2)** Implement **getActiveSigningKeys() function** which will return an array of active signing keys that became active after validators started.
_Current solution_: right now we can only check that added signing keys are started validation by check sp1.getUsedSigningKeys().
_Reasons_: To match used signing keys with active validators pubKeys in the eth2 side. We can  clearly check  sp1.usedSigningKeys() count,but can`t correctly compare usedSigningKeys  active validators pubKeys in eth2.

In conclusion, I might admit that in bundle sp signing keys -> to eth2 we can`t track correctly this action.
